app/models/guide.rb

```diff
@@ -0,0 +1,36 @@
+require "gds_api/publishing_api"
+
+class Guide < ActiveRecord::Base
+  validates :content_id, presence: true, uniqueness: true
+  validates :slug, presence: true
+  validates_associated :latest_edition
+
+  has_many :editions
+  has_one :latest_edition, -> { order(created_at: :desc) }, class_name: "Edition"
+
+  accepts_nested_attributes_for :latest_edition
+
+  PUBLISHERS = {
+    "Design Community" => "http://sm-11.herokuapp.com/designing-services/design-community/",
+    "Agile Community" => "http://sm-11.herokuapp.com/agile-delivery/agile-community"
+  }
+
+  before_validation on: :create do |object|
+    object.content_id = SecureRandom.uuid
+  end
+
+  after_save :publish
```

Comment:
Should we do something like [this](https://github.com/alphagov/policy-publisher/blob/c50215bc48a8e5894bd905e58b99a12abf2a5922/app/controllers/policies_controller.rb#L29) instead? We haven't merged this yet but already have had issues with callbacks in tests https://github.com/alphagov/service-manual-publisher/commit/7211d15e23f9bd51c792aa6d87778f50c7cb51c0.